charts/karmada/templates/karmada-apiserver.yaml

```diff
@@ -28,6 +28,18 @@ spec:
     spec:
       {{- include "karmada.apiServer.imagePullSecrets" . | nindent 6 }}
       automountServiceAccountToken: false
+      affinity:
```

Comment:
Why change karmada-apiserver?

Reply:
I want to unify the usage.

Reply:
I think the former which has better scalability is better.